src/helpers/from-spec-observable.ts

```diff
@@ -0,0 +1,39 @@
+import { observable } from '@legendapp/state';
+
+interface SpecObserver<T = unknown, E = unknown> {
+    next: (value: T) => void;
+    error: (error: E) => void;
+    complete: () => void;
+}
+
+interface SpecSubscription {
+    unsubscribe(): void;
+    closed?: boolean;
+}
+
+interface SpecObservable<T> {
+    subscribe: (observer: SpecObserver<T, any>) => SpecSubscription;
+}
+
+export function fromSpecObservable<T = unknown>(specObservable: SpecObservable<T>, initialValue?: T) {
+    const obs = observable<T>(initialValue);
+    const { unsubscribe } = specObservable.subscribe({
+        next: (value) => {
+            // I don't think this is the correct way to do this...
+            // I think this might need to be a call to `mergeIntoObservable`,
+            // but I will ask Mr. Meistrich
```

Comment:
I think this is right. We would want it to be exactly the same object, whereas mergeIntoObservable would merge the new value into the old value, which would be incorrect.